js/main.js

```diff
@@ -0,0 +1,17 @@
+const nav = document.querySelector('.nav-mobile');
+const navBtn = document.querySelector('.burger-btn');
+const allNavItems = document.querySelectorAll('.nav-mobile__item');
+
+const openNav = () => {
+	nav.classList.toggle('nav-mobile--active');
```

Comment:
This line looks like it's out of order. I believe it could be fixed easily by reformatting the file

Reply:
Does toggling `nav-mobile` class also work on non-mobile versions of your page?